### DISCUSSION TITLE
Any update on pingot?

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#119] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I saw someone is making a workaround using html,does anyone know how that's coming along?

**Comment by Alex2Mayhem** [comment]
> I saw someone is making a workaround using html,does anyone know how that's coming along?

 And who was that?

**Reply by Huntlb** [reply]
Holding The Creator

**Reply by Huntlb** [reply]
Sadly it hasn't been updated in a week

**Reply by ghost** [reply]
It'll be fine

**Reply by The-Man-Is-Me** [reply]
Yall just chill, it takes time to program. Maby if you offer a insentive someone will actually try to get one done fast, otherwise be patient.

**Reply by ghost** [reply]
@The-Man-Is-Me as a dev, we aren't dogs that want treats. It won't make us go any faster, it will just stress us out more

**Comment by 3kh0** [comment, marked as answer]
Just wait, you can wait a little longer.
